templates/common/infra/bicep/core/host/container-apps-environment.bicep

```
@@ -12,7 +12,7 @@ param daprEnabled bool = false
 @description('Name of the Log Analytics workspace')
 param logAnalyticsWorkspaceName string
 
-resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2023-04-01-preview' = {
+resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2023-04-01' = {
```

Should this one be 2023-05-01 as well?
```suggestion
resource containerAppsEnvironment 'Microsoft.App/managedEnvironments@2023-05-01' = {
```